web-ng/src/app/components/map/map.component.ts

```diff
@@ -128,10 +133,11 @@ export class MapComponent implements AfterViewInit, OnDestroy {
 
   private onProjectAndFeaturesUpdate(
     project: Project,
-    features: List<Feature>
+    features: List<Feature>,
+    selectedFeatureId: string | null
```

Comment:
Rather than pass selectedFeatureId around let's just get it from the NavigationService.